### DISCUSSION TITLE
GUI issue with "Replace and Play"

### DISCUSSION THREAD

**Discussion by Roland-B** [rern/rAudio#1217] [Fixed Issues] [answered]
@rern GUI issue with Albums, Webradio.. once an album or radio station is selected already and you scroll down a bit on the screen to choose an album or radio station the "Replace current playlist?" confirmation dialog box is missing and a large black gap appears (you can press [Enter] to continue with the confirmation buttons missing). Video attached.

<img src="https://user-images.githubusercontent.com/43496391/215586874-14dfb8cd-f78d-492b-a30b-84f8b0903978.mov" width=300>

Roland.

**Comment by hjfisch** [comment]
This also happens if you want to replace  Radio-Thumbnails in longer radiolists ( more than 20-25 Stations) in the last ones.
To avoid this ,I've made some Subdirectories in the Webradio Directory with not more than 20 Stations .

**Comment by hjfisch** [comment]
It seems to be a windows problem.
It's working in Linux and Android.

**Reply by Roland-B** [reply]
@hjfisch. Nope, sadly not just a Windows problem for me, it is also broken on my android device, and Linux (Mint). @rern Have also tested on latest image [i20230130](https://github.com/rern/rAudio-1/releases/tag/i20230130) and is the same.

@hjfisch If you mean for your issue with thumbnails, then can you kindly raise a seprate bug report please if you haven't already, otherwise may confuse the issue I have reported here. Thanks in advance.

Android phone recording

<img src="https://user-images.githubusercontent.com/43496391/215822968-e87d92dd-a83b-4c6a-894a-e3eadfa62baa.mov" width=300>

Roland B.

**Comment by hjfisch** [comment]
I am also on Linux Mint and for me this bug is only on  Windows.B.t.w I use Waterfox and Vivaldi on Mint.I've recognized this behaviour first by replacing thumbnails , but it also happens if I want to replace in playlist.

**Reply by Roland-B** [reply]
@hjfisch. Also tried on Linux Mint using Waterfox with fresh image rAudio image (20230130) and the issue is exactly the same. Only thing I can think of is that you may not be replicating the issue the same as me (you probably don't have enough stations / long enough list on the screen to replicate similarly?? as you mentioned you have moved them to folders to stop the thumbnail issue). Who knows?!

I have also tried the change cover down the bottom of my list of 70 stations and get the same error as you mentioned.

@rern.  Can confirm the issue is also there for "change cover art" as "hjfisch" first mentioned.

**Comment by LoacBZH** [comment]
Hi!
Same problem with a 7' touchscreen, in local ui. But here, I can't reload the page, so need to reboot if I have this problem

**Comment by Roland-B** [comment, marked as answer]
Fixed in update [20230202](https://github.com/rern/rAudio-1/releases/tag/20230202). Cheers.
